### ISSUE TITLE
[RequestAddOwner] Exception handling screen

### ISSUE THREAD

**Issue by DmitryBespalov** [safe-global/safe-ios#2347]
Exception handling screen is shown when a user opens the 'add owner' link but there's no such safe in the app, or if the safe exists but the current user is not an owner of the safe, or if the account already added to the safe.

https://www.figma.com/file/2EgEwJHWRBx7KdGdPRezgn/Owner-Keys?node-id=1377%3A15806

https://www.figma.com/file/2EgEwJHWRBx7KdGdPRezgn/Owner-Keys?node-id=1407%3A15385

https://www.figma.com/file/2EgEwJHWRBx7KdGdPRezgn/Owner-Keys?node-id=2016%3A15439

![image.png](https://images.zenhubusercontent.com/5c361cfa7727940fef83693b/03906bf0-c467-4600-8593-38b57e79ecba)

## What to do
1. Create the UI for the screens
2. Implement the flow for the respective conditions: a) when no safe exists, b) when there's no owner of that safe c) when no safe and no owner of that safe, d) when safe already an owner

![image.png](https://images.zenhubusercontent.com/5c361cfa7727940fef83693b/e17b8f9e-68ef-44ad-8149-4609bcbd579b)

## Acceptance criteria
1. Screen is shown when opening the 'add owner' link and there's no safe or it is read-only
1. If account already an owner of the safe, then show that link is not active anymore.
1. If no safe exists, then this screen suggests to add safe. Then if there's no owner, it suggests to add it.
1. If there's safe and no owner, then suggests to add owner key. When added successfully, then user sees the correct "start" screen that handles the 'add owner' URL.

**Comment by DmitryBespalov** [comment]
@DmitryBespalov add the "already added" case

**Comment by DmitryBespalov** [comment]
Test cases that I handled:

- Safe does not exist on a blockchain
  - https://gnosis-safe.io/app/eth:0xDF9e7F58bE74Ff111c7c06000971cF1E10B92dD8/addOwner?address=0x5f78fBaB81f9892bbe379d88c8a224774411B0a9 
- Safe exists, not added to the app, and the owner is already in the list
  - https://gnosis-safe.io/app/eth:0xfF501B324DC6d78dC9F983f140B9211c3EdB4dc7/addOwner?address=0x9F87C1aCaF3Afc6a5557c58284D9F8609470b571 
- Safe not added in the app, exists, owner is new
- Safe added but it is read-only, owner is new
- Safe added, owner added, owner is new


Tried on Ethereum and on Rinkeby.
Tried when there are no safes in the app.
Tried when the safe is not selected.
